dnsapi/dns_dynv6.sh

```diff
@@ -16,8 +16,8 @@ dynv6_api="https://dynv6.com/api/v2"
 # Please Read this guide first: https://github.com/Neilpang/acme.sh/wiki/DNS-API-Dev-Guide
 #Usage: dns_dynv6_add  _acme-challenge.www.domain.com  "XKrxpRBosdIKFzxW_CT3KLZNf6q0HG9i01zxXp5CPBs"
 dns_dynv6_add() {
-  fulldomain=$1
-  txtvalue=$2
+  fulldomain=$(echo "$1" | tr '[:upper:]' '[:lower:]')
+  txtvalue="$2"
```

Comment:
please use `_lower_case()`

Reply:
Sorry I overlooked the existence of that function but this is changed now.

Also thanks for the quick response!